src/main/java/net/dv8tion/jda/api/requests/Route.java

```diff
@@ -14,52 +14,40 @@
  * limitations under the License.
  */
 
-package net.dv8tion.jda.internal.requests;
+package net.dv8tion.jda.api.requests;
 
 import net.dv8tion.jda.internal.utils.Checks;
+import net.dv8tion.jda.internal.utils.EncodingUtil;
 import net.dv8tion.jda.internal.utils.EntityString;
 import net.dv8tion.jda.internal.utils.Helpers;
 
 import javax.annotation.CheckReturnValue;
 import javax.annotation.Nonnull;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.StringJoiner;
 
-import static net.dv8tion.jda.internal.requests.Method.*;
+import static net.dv8tion.jda.api.requests.Method.*;
 
+/**
+ * Routes for API endpoints.
+ */
 @SuppressWarnings("unused")
 public class Route
 {
     public static class Misc
     {
-        public static final Route TRACK =             new Route(POST, "track");
         public static final Route GET_VOICE_REGIONS = new Route(GET,  "voice/regions");
         public static final Route GATEWAY =           new Route(GET,  "gateway");
         public static final Route GATEWAY_BOT =       new Route(GET,  "gateway/bot");
     }
 
     public static class Applications
     {
-        // Bot only
         public static final Route GET_BOT_APPLICATION =             new Route(GET, "oauth2/applications/@me");
         public static final Route GET_ROLE_CONNECTION_METADATA =    new Route(GET, "applications/{application_id}/role-connections/metadata");
         public static final Route UPDATE_ROLE_CONNECTION_METADATA = new Route(PUT, "applications/{application_id}/role-connections/metadata");
-
-        // Client only
-        public static final Route GET_APPLICATIONS =              new Route(GET,    "oauth2/applications");
-        public static final Route CREATE_APPLICATION =            new Route(POST,   "oauth2/applications");
-        public static final Route GET_APPLICATION =               new Route(GET,    "oauth2/applications/{application_id}");
-        public static final Route MODIFY_APPLICATION =            new Route(PUT,    "oauth2/applications/{application_id}");
```

Comment:
I don't think these are client only anymore as we've opened up the ability of an application to modify itself.
Not sure if that's documented yet, but it's coming.

Reply:
These are specifically to get other applications by ID. I'd expect an application modifying itself would use `@me` instead of an id.